gqlalchemy/how-to-guides/networkx.md

```diff
@@ -0,0 +1,96 @@
+---
+id: networkx
+title: How to import NetworkX graph into Memgraph
+sidebar_label: Import NetworkX graph into Memgraph
+---
+
+GQLAlchemy holds transformations that allow the NetworkX graph to be transformed into the Memgraph graph. These transformations take the NetworkX graph object and translate it to the appropriate Cypher queries. Then the Cypher queries can be executed so that the graph is created inside Memgraph. 
+
+
+## Import NetworkX graph into Memgraph 
+
+### 1. Run Memgraph
+
+To be able to try out this example on your own, you need a running Memgraph instance. Run Memgraph using the Memgraph Platform Docker image, which includes both the MAGE library and Memgraph Lab, a visual interface. 
+To run the image, open a command-line interpreter and run the following Docker command:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 -p 3000:3000 memgraph/memgraph-platform:latest
+```
+
+### 2. Create and run a Python script
+
+Create a new Python script `graph.py`, in the code editor of your choice, with the following code:
+
+```python
+import networkx as nx
+from gqlalchemy import Memgraph
+from gqlalchemy.transformations import nx_to_cypher
+
+memgraph = Memgraph()
+memgraph.drop_database()
+graph = nx.Graph()
+graph.add_nodes_from([(1, {"labels": "First"}), (2, {"name": "Kata"}), 3])
+graph.add_edges_from([(1, 2, {"type": "EDGE_TYPE", "date": "today"}), (1, 3)])
+
+for query in nx_to_cypher(graph):
+    memgraph.execute(query)
+```
+
+To run it, open a command-line interpreter and run the following command:
+
+```python
+python3 graph.py
+```
+
+### 3. Explore the graph
+
+[Connect to Memgraph](/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph) via Memgraph Lab which is running at `localhost:3000`. Head over to the **Query Execution** tab and write the following query:
+
+```cypher
+MATCH (n)-[r]->(m)
+RETURN n, r, m;
+```
+
+Click **Run Query** button to see the results.
+
+<img src={require('./data/networkx-example-2.png').default} alt="networkx-example-1" className={"imgBorder"}/>
+
+You can notice that the node with the property `name` Kata and property `id` 2 doesn't have a label. This happened because we didn't define property key `labels`. On the other hand, a relationship which doesn't have a key `type` in the properies dictionary, is of type `TO`.
```

Comment:
```suggestion
You can notice that the node with the property `name` Kata and property `id` 2 doesn't have a label. This happened because the property key `labels` was not defined. A relationship without a key `type` in the properties dictionary is of type `TO`.
```